consensus/reactor_test.go

```diff
@@ -11,6 +11,9 @@ import (
 	"testing"
 	"time"
 
+	"github.com/tendermint/tendermint/crypto/ed25519"
```

Comment:
can you place these after 3rd party testify imports?

Reply:
yes